component.js

```diff
@@ -257,15 +265,21 @@ function factory (initialOptions) {
        * @returns {ReactElement}
        * @api public
        */
-      var create = function (keyOrProps, propsOrPublicContext, ReactUpdateQueue) {
-        // After stateless arrow functions was allowed as components, react will instantiate
-        // the `create` function if it has a prototype. We are passed `props`, `publicContext`
-        // and `ReactUpdateQueue`.
-        // https://github.com/facebook/react/blob/88bae3fb73511893519195e451c56896463f669b/src/renderers/shared/reconciler/ReactCompositeComponent.js#L154-L171
-        if (typeof ReactUpdateQueue == 'object' && !_isNode(ReactUpdateQueue)) {
+      var create = function create (keyOrProps, propsOrPublicContext, updater) {
```

Comment:
The function now being named, we might as well make it work without `var create`?

Its used a bit further down for the following, but I see no real use for the assignment..

```js
  if (methodStatics) {
    create = assign(create, methodStatics);
  }
```